Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+torch = "*"
+Pillow = "*"
+mmc = {editable = true, path = "."}
+clip-anytorch = "*"
```

Comment:
I'd recommend specifying a major version here

Reply:
could you recommend a minimal major version for pillow and/or torch? I keep getting into trouble setting versions for those packages in mmc specifically. I especially want to make sure installing mmc doesn't trigger uninstall of a working gpu-torch and replacing it with a cpu only version.